tensorboard/components/experimental/plugin_util/plugin_api_host_test.ts

```diff
@@ -246,17 +244,23 @@ describe('plugin_api_host test', () => {
               triggerGetUrlData = callback;
             }
           );
-
-        getUrlDictSpy = spyOn(tf_storage_utils, 'getUrlDict');
       });
 
       it('returns url data from the tf storage', () => {
-        getUrlDictSpy.and.returnValue({
-          globalThing: 'hey',
-          'p.plugin_id.a': '1',
-          'p.plugin_id.b': 'b',
-          'p.another_plugn.b': '2',
-        });
+        // Do not rely on Polymer bundle in the test.
+        const createElementSpy = spyOn(document, 'createElement');
+        createElementSpy.withArgs('tf-storage').and.returnValue({
+          tf_storage: {
+            getUrlHashDict: () => {
+              return {
+                'globalThing': 'hey',
+                'p.plugin_id.a': '1',
+                'p.plugin_id.b': 'b',
+                'p.another_plugn.b': '2',
+              }
+            },
+          },
+        } as any);
```

Comment:
Is there a reason not to use the tb_polymer_interop_type and undo `any` here?

Reply:
Switched to `as unknown as TfStorageElement`.

When trying to directly cast/declare a `TfStorageElement`, TS complains that our plain object is too different from `HTMLElement`, so it requires the unknown first.  This unknown casting loses actual typechecking benefits, but at least it is clearer to readers what type it is.